tests/test_nb_html.py

```diff
@@ -12,7 +12,7 @@
 def test_init_repo(tmpdir):
     with zipfile.ZipFile(str(Path(__file__).parent / 'repo_data.zip'), 'r') as zip_ref:
         zip_ref.extractall(tmpdir)
-    pytest.raw_htmls = Path(tmpdir / 'raw')
+    pytest.raw_htmls = Path(tmpdir) / 'raw'
```

Comment:
I've never seen this technique of assigning new attributes to `pytest` - basically creating globals - to pass around state between tests. It also relies on tests being executed in a particular order. I think this is what [fixtures](https://docs.pytest.org/en/stable/fixture.html) are for.

Reply:
I've tried `fixtures` in conftest.py but then for each single test in all test files the repo will be unzipped etc. Is there a way to tell pytest to do it only for tests inside test_nb_html?

Reply:
Also, `glottolog_repo` is just 164K - and that includes a bunch of `__MACOSX` stuff. So I'd say there is no need to zip it at all.

Reply:
Typically, `assert`s in tests may change together with some data in `glottolog_repo` - so having one zipped and thus obscuring this correspondence would be too intransparent.